### PR TITLE
Milestone: Peers are able to connect to bootstrap

### PR DESCRIPTION
**Pull request by adaszko**

### CODE REVIEW

File: hardnet1/main.tf [hunk 1]
```diff
@@ -249,3 +249,56 @@ resource "google_compute_instance" "node" {
...
+    protocol = "tcp"
+    ports = [ 22, 40403 ]
+  }
+}
+
+resource "google_compute_firewall" "fw_public_node_rpc" {
+  name = "${var.resources_name}-node-rpc"
+  network = "${data.google_compute_network.default_network.self_link}"
+  priority = 540
+  target_tags = [ "${var.resources_name}-node" ]
```
Comment (woky) [inline hardnet1/main.tf]: This is OK for development or public network, but later, when we're not deploying to the network from our PCs, it'd be good make only the orchestrator machine privileged to use gRPC (and deploy) via `source_tags`. Before that, this https://github.com/rchain/hardening/blob/master/hardnet1/main.tf#L155 should be changed to e.g. `${var.resources_name}-orchestrator`.

File: hardnet1/main.tf [hunk 2]
```diff
@@ -249,3 +249,56 @@ resource "google_compute_instance" "node" {
...
+    protocol = "tcp"
+    ports = [ 40401 ]
+  }
+}
+
+resource "google_compute_firewall" "fw_node_p2p" {
+  name = "${var.resources_name}-node-p2p"
+  network = "${data.google_compute_network.default_network.self_link}"
+  priority = 550
+  source_ranges = [ "0.0.0.0/0" ]
```
[the remark this answers is not in the document]
Comment (adaszko) [reply hardnet1/main.tf]: I think it can be done now.

[the remark this answers is not in the document]
Comment (adaszko) [reply hardnet1/main.tf]: OK, done.

File: hardnet1/main.tf (hunk 1 above)
Comment (adaszko) [reply hardnet1/main.tf]: Done as well.

**Pull request merged**